### PR TITLE
LTP: Update fanotify14 for 15.5

### PR DESCRIPTION
**Pull request by dzedro**

### CODE REVIEW

File: ltp_known_issues.yaml [hunk 1]
```diff
@@ -6,9 +6,10 @@ syscalls:
       retval: ^1$
       message: Test bug. Reported to upstream LTP. bsc#3541571
     fanotify14:
-    - product: opensuse:Tumbleweed|opensuse:15.4
+    - product: opensuse:(Tumbleweed|15\.[45])
+      kernel: ^5
```
Comment (mdoucha) [inline ltp_known_issues.yaml:10]: Adding a `kernel` condition will break this entry for Tumbleweed.

**Review by mdoucha — CHANGES_REQUESTED** [review]

File: ltp_known_issues.yaml (hunk 1 above)
Comment (dzedro) [reply ltp_known_issues.yaml:10]: The issue is not present on TW anymore

Comment (mdoucha) [reply ltp_known_issues.yaml:10]: OK, then Tumbleweed should be removed and the kernel filter is not needed. opensuse 15.4/15.5 will stay on kernel 5.x until EOL.

Comment (dzedro) [reply ltp_known_issues.yaml]: Great, done.

**Comment by pevik** [comment]
Thank you both.

**Pull request closed without merging**